spec/models/repository_owner/base_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe RepositoryOwner::Base do
+  context ".sanitize_response_hash" do
+    let(:hash) do
+      {
+        "id" => 1234,
+        "name" => "My \u0000Name With\u0000 Null Bytes",
+        "company" => "My \u0000Company",
+      }
+    end
+    let(:result) { RepositoryOwner::Base.sanitize_response_hash(hash) }
+
+    it "should return a HashWithIndifferentAccess" do
+      expect(result).to be_a(HashWithIndifferentAccess)
```

Comment:
🎱 Any particular reason why the output of a method named `sanitize.....hash`  needs to have indifferent access...? Should it be named something like `sanitize_and_transform_to_indifferent_access_hash`? Should this also deal with the url/utf-8 escaping etc.? (that's what I think of when I see `sanitize_...` in a method name)

Reply:
yeah, I can definitely make the method name more specific. I don't think we should add encoding logic here until we need it though (just to keep it simple).